### ISSUE TITLE
The README and tutorial pages may need updating

### ISSUE THREAD

**Issue by MathiasMalandain** [ghorwin/FMICodeGenerator#45]
# Description

I started using FMICodeGenerator by following the instructions on [this page](https://github.com/ghorwin/FMICodeGenerator/wiki/Test-scenario:-Stateless-P-controller), on a Lubuntu 22.04 (based on Ubuntu 22.04.2 LTS "Jammy") virtual machine. Several steps from this tutorial, as well as the `README` file itself, could not be performed, or not "as is".

## Wrong dependencies

Everything I needed to make `FMICodeGenerator` work was already installed. However, I noticed several errors/discrepancies when trying to run [the setup command for Python 3.x](https://github.com/ghorwin/FMICodeGenerator#linux):

* `sudo apt get install ...` will not work. The word `get` should be removed.
* `built-essential` cannot be found. You likely meant `build-essential`.

Instructions on Ubuntu 21.04 or newer may be added: `qt5-default` [is obsolete since at least 21.04](https://askubuntu.com/questions/1335184/qt5-default-not-in-ubuntu-21-04).

## A-Z, a-z only

When [declaring variables](https://github.com/ghorwin/FMICodeGenerator/wiki/Test-scenario:-Stateless-P-controller#variables), I got a message warning me that only ASCII characters A-Z and a-z could be used in variable names. I had to rename variables `T_limit` and `k_P` accordingly. Was it unobserved behavior so far? I would not expect so, as the warning message makes it really clear which characters can be used. The ability to use underscores in variable names would be hugely convenient.

## No compliance checker

Command `fmuCheck.linux64`, [used here](https://github.com/ghorwin/FMICodeGenerator/wiki/Test-scenario:-Stateless-P-controller#test-the-fmu-with-the-compliance-checker), is not known to my VM, and I do not think that either the tutorial or the `README` file says anything about it. I will be exporting the FMU from another tool to make sure it works.

**Comment by ghorwin** [comment]
Hi Mathias,

thanks for the feedback. Indeed, some things need to be fixed/updated, and I'll try to get this done next weekend. 

*Az/az issue:* in a previous version I didn't have a check at all and people tried to use all kinds of characters in names that did not compile in C (like / or \ or & or | ...). I didn't have the time to do a proper syntax check for variable names that yield valid C++ variable names, so I restricted the variables to use letters only. Of course, _ is a very frequently used character in variables, so I should add this; same as numbers, unless the variable starts with a number (forbidden in C/C++).

The dependency section should be made distro-version dependend, and of course it is either "apt install" or "apt-get install". I'll fix this also.

Regarding the FMU checker, it is provided in a zip archive (and also from the FMI tools webpage) and needs to be extract into a binary search path (either system wide or locally with updated PATH variable in your .bashrc).

Once I've incorporated the changes in the doc/python code, I'll close the issue.
